packages/plugin-nested-docs/src/hooks/resaveChildren.ts

```diff
@@ -115,7 +115,7 @@ export const resaveChildren =
     await resave({
       collection,
       doc,
-      draft: doc._status === 'published' ? false : true,
+      draft: (!doc._status || doc._status === 'published') ? false : true,
```

Comment:
I would typically check the collection config to see if versions.drafts is enabled. That said, I see how this would work also.
Good job!